### PR TITLE
Run multiple node servers in one terminal and one command with concurrently + Refactor URIs into constants

### PR DESCRIPTION
**Pull request by TheSpaceCuber**
* Updated README with instructions on how to use it. Not a dependency since can install globally.
* Since all node servers are in the same terminal, require unique print statements for index.js.
* Update gitignore
* Changed all ENV to NODE_ENV
* Abstract env variables into constants.js

Also, it turns out that for index.js you should import dotenv before routes if not constant.js will break. Alternatively you can import dotenv in constants.js also.

To test this you will need an env file in the root directory instead. Only that env file is need, rest are useless.

### CODE REVIEW

File: matching-service/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import 'dotenv/config'
 
 let uri;
 
-if (process.env.ENV == "PROD") {
+if (process.env.NODE_ENV == "production") {
     uri = process.env.DB_CLOUD_URI
 } else if (process.env.DB_DOCKER_URI) {
     uri = process.env.DB_DOCKER_URI
```
Comment (ongweisheng) [inline matching-service/index.js:12]: Make equal changes to all the other service, NODE_ENV == "production" instead of ENV == "PROD"

**Review by ongweisheng — CHANGES_REQUESTED** [review]

**Review by ongweisheng — APPROVED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -27,16 +29,40 @@ npm start
...
+
+Ensure that either a mongo server is up or the connection string points to Atlas. See below for instructions on setting up a local mongo container.
+
+First you have to create a `.env` file in the root folder. It should look like the following:
+
+```
+DB_CLOUD_URI=
+DB_LOCAL_URI=mongodb://localhost:27017/cs3219
+DB_TEST_URI=mongodb://localhost:27017/userTests
+SECRET=
```
Comment (hiongkaihan) [inline README.md:43]: Maybe can add a note/comment under the env example like "add DB_CLOUD_URI if you want to connect to mongodb cloud....SECRET if you want to set the secret variable..."

or

add `<Placeholder>` for DB_CLOUD_URI and SECRET 
eg.  SECRET=`<secret phrase for token encryption>` DB_CLOUD_URI=`<MongoDB cloud connection string>`

**Review by hiongkaihan — APPROVED** [review]
LGTM

**Pull request merged**